### PR TITLE
Update capacities - AT, EE, HU, LV, LT, RO

### PR DESCRIPTION
**Pull request by ThierryOllivero**

### CODE REVIEW

**Comment by ThierryOllivero** [comment]
Partially solves issue #66

**Comment by corradio** [comment]
Why partially?

**Comment by ThierryOllivero** [comment]
Answering to your question @corradio : "Why partially" : #66 is for "All Countries". This PR only deals with a few countries (the ones added today). I consider #66 to be an open issue, on which we will come back each time we need to add data for a country. Would that be ok?

**Comment by corradio** [comment]
Mmh let's avoid to have "always open" issues. I'd prefer updating #66 to specifically say which countries are lacking so we can close it when done (like a task).

**Comment by ThierryOllivero** [comment]
@corradio I also changed the Readme. In the end, I kept your data for existing countries, and added new countries.

File: api/static/app/countryconfig.js [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
     countries['DE'].data.capacity = {
-        biomass: 8970,
-        coal: 28310 + 21140,
-        gas: 28490,
-        hydro: 5590,
-        nuclear: 10790,
-        oil: 4240,
-        solar: 39630,
-        wind: 3740 + 42460,
+        biomass: 6609 + 1581,
```
Comment (corradio) [inline api/static/app/countryconfig.js]: Germany was modified but the README was not modified

File: api/static/app/countryconfig.js [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+        coal: 28670 + 22015,
+        gas: 26694 + 73,
+        hydro: 8644 + 3749 + 539,
+        nuclear: 10952,
+        oil: 3745,
+        solar: 38994,
+        wind: 3256 + 39937,
     };
     countries['DK'].data.capacity = {
+        biomass: 197,
```
Comment (corradio) [inline api/static/app/countryconfig.js]: Nice! Can you update the readme with where the Danish data came from?

**Comment by corradio** [comment]
Thanks buddy!

**Pull request merged**